### PR TITLE
fix: animation update

### PR DESCRIPTION
**Pull request by RoChernikov**
CSS-only animation for illustration in the red slide (pro) for the best performance (much smoother). JS-free.

**Video demonstration** (Safari 15.3 (17612.4.9.1.5), ProMotion Display 120hz)

https://user-images.githubusercontent.com/84858680/152639198-ed5a906a-b0a5-4573-a653-a7ad3df14acc.mov

### CODE REVIEW

**Comment by RoChernikov** [comment]
[Video demonstration](https://youtu.be/pw2ojx4GSZc)

File: src/styles/slide.css [hunk 1]
```diff
@@ -342,23 +342,36 @@
...
+    display: grid;
+    align-content: center;
     width: calc(50vw - ((100vw - 100%)/2)); /* use viewport because of the fixed position (percentage could be incorrect); https://stackoverflow.com/a/34884924 */
     box-sizing: border-box;
     border-right: solid 1px $grey-text;
 
+    &--sticky-container {
+      position: relative;
+      width: 100%;
+      height: 1600px;
```
[the remark this answers is not in the document]
Comment (RoChernikov) [reply src/styles/slide.css]: Sure, done! Seems perfect now. 🙌🏻

File: src/slides/pro.pug [hunk 2]
```diff
@@ -3,8 +3,9 @@
     include /images/pro-badge-no-free.svg
 .slide.red.no-border#pro
   .slide--illustration.desktop-only
-    .slide--illustration--content.js-slide-illustration
-      include /images/logo-pro.svg
+    .slide--illustration--sticky-container
+      .slide--illustration--content.js-slide-illustration
```
[the remark this answers is not in the document]
Comment (RoChernikov) [reply src/slides/pro.pug]: Almost forgot, thanks! Done. 👌🏻

**Comment by palkan** [comment]
Thank you!

**Pull request merged**